src/Functions/generateSerialID.cpp

```diff
@@ -91,7 +91,7 @@ class FunctionSerial : public IFunction
     DataTypePtr getReturnTypeImpl(const ColumnsWithTypeAndName & arguments) const override
     {
         FunctionArgumentDescriptors mandatory_args{
-            {"series name", static_cast<FunctionArgumentDescriptor::TypeValidator>(&isString), nullptr, "String"}
+            {"series_identifier", static_cast<FunctionArgumentDescriptor::TypeValidator>(&isString), &isColumnConst, "const String"}
```

Comment:
I know you mentioned in the description changes in [`fba1fc4` (#74114)](https://github.com/ClickHouse/ClickHouse/pull/74114/commits/fba1fc406257b9ffe21d088f393f45599c934e94)  are unrelated, but since they're within this PR, care to explain a little bit the reasoning behind this and the implications?

Reply:
Sure. After new year, I did a bunch of "post-morten" (lol) reviews, meaning reviews after PRs have been merged. One of the PRs was #73950. I noticed a few minor inconsistencies and fixed them:
- the include order,
- naming of identifiers in `mandatory_args`,
- the docs (l. 170ff),
- usage of `&isColumnConst` in `mandatory_args` which is a declarative way of checking for constness (so the existing check could go)

There are really no implications except that the code is a little bit more consistent. And because the change was small, I didn't want to make a separate PR for it.

Reply:
Got it, merge it freely whenever you deem so